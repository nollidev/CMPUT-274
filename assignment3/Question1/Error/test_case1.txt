Traceback (most recent call last):
  File "/home/nattyeddy/Documents/School/Uni Homework/Second Year/CMPUT-274/assignment3/Question1/solutionQuestion1.py", line 41, in <module>
    main()
    ~~~~^^
  File "/home/nattyeddy/Documents/School/Uni Homework/Second Year/CMPUT-274/assignment3/Question1/solutionQuestion1.py", line 37, in main
    read_tree().print_tree()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/nattyeddy/Documents/School/Uni Homework/Second Year/CMPUT-274/assignment3/Question1/treenode.py", line 15, in print_tree
    children_values = [c.get_value() for c in self.__children]
                       ^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'get_value'
