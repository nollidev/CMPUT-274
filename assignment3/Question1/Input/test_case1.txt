12 3
W A B X C - - D E F G H
