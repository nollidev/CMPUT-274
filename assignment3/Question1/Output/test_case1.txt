['W', 'A', 'B', 'X', 'C', '-', '-', 'D', 'E', 'F', 'G', 'H']
