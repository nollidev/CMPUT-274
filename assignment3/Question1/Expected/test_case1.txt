W -> A B X
A -> C
B -> D E F
X -> G H
